src/git.ts

```diff
@@ -80,11 +80,21 @@ export default class GitHub {
       token: token!
     });
 
-    this.logger.veryVerbose.info('Sucessfully authenticated with GitHub.');
+    this.logger.veryVerbose.info('Successfully authenticated with GitHub.');
 
     return Promise.resolve();
   }
 
+  // Looks like: https://developer.github.com/v3/repos/#get
+  public async getRepoMetadata(): Promise<any> {
```

Comment:
it looks like `this.ghub.repos.get` return a type of `GHub.Response<GHub.ReposGetResponse>` so the return type for this function should probably be `Promise<GHub.Response<GHub.ReposGetResponse>>`

Reply:
```suggestion
  public async getRepoMetadata(): Promise<GHub.Response<GHub.ReposGetResponse>> {
```